deployment/prod/convo-server/convo.Dockerfile

```diff
@@ -10,17 +10,20 @@ WORKDIR /app
 # Copy the requirements file into the container
 COPY requirements.txt .
 
+#install uv
+RUN pip install uv
```

Comment:
why pip install uv here and curl -LsSf https://astral.sh/uv/install.sh | sh
above?

Reply:
@kinshuksinghbist please answer this